CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,16 @@
+Update Docker security practices (Add HEALTHCHECK, Use Anonymous User, Use two-stage build)
+Fix: Force finish transaction after process a device measure
+Fix: Do not intercept error about DEVICE_NOT_FOUND in findOrCreate device (iotagent-node-lib#889)
+Fix srv, subsrv, transaction and correlator id in logs of http binding
+Fix some log levels and details at bindings
+Overall update of dev package dependencies
+Error processing zero measures (#486)
+Update codebase to use ES6
+-  Remove JSHint and jshint overrides
+-  Add esLint using standard tamia presets
+-  Replace var with let/const
+-  Fix or disable eslint errors
+Set Nodejs 10 as minimum version in packages.json (effectively removing Nodev8 from supported versions)
+Fix the writing of same correlator and transaction id in logs (issue #426). 
+Add: config.mqtt.avoidLeadingSlash flag (IOTA_MQTT_AVOID_LEADING_SLASH) to avoid leading slash in MQTT
+Add: explicitAttrs flag (in configuration and also group/device provisioning) to progress only the measures corresponding to attributes declared in attributes array (#416)
```

Comment:
```suggestion
Update Docker security practices (Add HEALTHCHECK, Use Anonymous User, Use two-stage build)
```

which I understand with the work in this PR

Reply:
Fixed e2b3296